### PR TITLE
Using a Serverless Architecture to deliver IRC Webhook Notifications

### PR DESCRIPTION
**Pull request by marvinpinto**
**Thank you for reading this!**

I would be very grateful for your feedback (should you choose).
## Things to ignore

I will rebase and adjust the dates of the post right before I publish (merge) this. You can ignore that for now!
## How do I provide feedback?

I trust you to use the right mechanism to provide me feedback. If you are okay with publicly commenting on this PR, I'm game :+1:.

If you would rather choose to get in touch with me privately, that works for me too!

IRC or email work best for me, if possible. (`marvinpinto` on freenode)
## Thank you again!

You rock :guitar:

### CODE REVIEW

File: content/post/2016-NN-NN-introducing-irc-hooky.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+itself with higher-order logic.
+
+In a traditional Amazon world with EC2 instances and autoscaling groups, the
+_physical_ hardware component has been abstracted out into virtual components,
+but infrastructure and resources are very much still a thing that
+the people using them need to actively manage.
+
+_A serverless world promises to be this magical land where
+network/infrastructure/scaling will **Just Work** and the only thing that will
+be an application developer's concern is the application itself._
```
Comment (hutchic) [inline content/post/2016-NN-NN-introducing-irc-hooky.md]: Described as it is above doesn't sound any different from a PaaS like heroku

File: spellcheck_ignore_words [hunk 2]
```diff
@@ -54,6 +66,11 @@ public's
 Rackspace
 reproducibility
 SaaS
+Schweet
```
Comment (hutchic) [inline spellcheck_ignore_words:72]: really? :P

**Comment by hutchic** [comment]
LGTM

**Comment by marvinpinto** [comment]
Thanks for reading it over @hutchic :smile:

**Pull request merged**